src/containers/CardContainer.js

```diff
@@ -1,10 +1,31 @@
 import React, { Component } from 'react';
 import Card from './../components/Card';
+import './Card.css'
+
+
 
 class CardContainer extends Component {
+
+  constructor (props) {
+    super(props)
+    this.state = {
+      flipped: false
+    }
+  }
+
+
+  cardFlip = (props) => {
```

Comment:
This could be name just `flip`

Reply:
Regarding the implementation, we can just toggle the state with something like this

```
this.setState(prevState => {
  return {flipped: !prevState.flip};
});
```

We can start adding some tests to this behavior, I will set-up Enzyme to start testing